lib/include/prjxray/xilinx/frames.h

```diff
@@ -67,13 +66,18 @@ int Frames<ArchType>::readFrames(const std::string& frm_file_str) {
 		std::vector<std::string> frame_data_strings =
 		    absl::StrSplit(frame_delta.second, ',');
 
-		if (frame_data_strings.size() != ArchType::words_per_frame) {
-			std::cerr << "Frame " << std::hex << frame_address
-			          << ": found " << std::dec
-			          << frame_data_strings.size()
-			          << " words instead of "
-			          << ArchType::words_per_frame << std::endl;
-			continue;
+		// Spartan6's IOB frames can have different word count
```

Comment:
Should each arch have a function that checks this?  On xc7, US and US+, it will be a straight forward check, and spartan6 will have a data lookup?

Reply:
What kind of check do you have in mind for Spartan6? Do you mean something like checking if the current frames is indeed an IOB frame and if not perform the same check as for all other architectures?

Reply:
Sort of.  Is the IOB frame fully variable, or just a different number?

Reply:
Let me rephrase.  Is there a way to write this check that is valid for all spartan 6 frame addresses?  Why not implement that check?

Reply:
Unsure. I just read that the IOB frame length can differ, but it didn't say exactly how and when. In my particular case it had something around 20 when the regular frames is 65. But I can't tell what the number is gonna be for other parts. So at this moment I can't really think of any generic check...

Reply:
Ok, let's file an issue to investigate and fix this later.

Reply:
Filed #1113.